src/ansys/tools/visualization_interface/backends/pyvista/pyvista.py

```diff
@@ -324,8 +315,10 @@ def hover_callback(self, _widget, event_name) -> None:
         renderer = plotter.iren.get_poked_renderer(x, y)
         self._hover_picker.Pick(x, y, 0, renderer)
         actor = self._hover_picker.GetActor()
-        if event_name == "TimerEvent" and actor is not None and actor in self._object_to_actors_map:
+        if actor is not None and actor in self._object_to_actors_map:
```

Comment:
Assuming that there is no key with the value ``None``, this is more clear:

```suggestion
        if actor in self._object_to_actors_map:
```

Furthermore, if you have a property called ``self.object_to_actors_map`` which points to ``self._object_to_actors_map``, you should use it internally as much as you can, aka use your own API. :)

Reply:
You are right, though the property is defined in `PyVistaInterface`, and I use it in here in `PyVistaBackendInterface`. I will use the API from `PyVistaInterface`, but I won't expose the attribute in `PyVistaBackendInterface` since it's not needed.